x-pack/legacy/plugins/epm/public/screens/detail/installation_button.tsx

```diff
@@ -4,20 +4,20 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 import React from 'react';
-
 import { EuiButton, EuiButtonEmpty } from '@elastic/eui';
 
 interface InstallationButtonProps {
-  isLoading: boolean;
-  isInstalled: boolean;
+  installationStatus: string;
```

Comment:
I believe this should be a type of `InstallStatus`. Otherwise any `string` is valid.